rubin_sim/maf/metrics/useMetrics.py

```diff
@@ -0,0 +1,65 @@
+# imports
+import numpy as np
+
+from rubin_sim.maf.metrics.baseMetric import BaseMetric
```

Comment:
Can you make this a relative import? 
from .baseMetric import BaseMetric
(I believe relative imports are preferred, but please comment if that's not correct)

Reply:
I've always understood the opposite. From PEP8: "Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) if the import system is incorrectly configured (such as when a directory inside a package ends up on sys.path)... However, explicit relative imports are an acceptable alternative to absolute imports, especially when dealing with complex package layouts where using absolute imports would be unnecessarily verbose:... Standard library code should avoid complex package layouts and always use absolute imports. Implicit relative imports should never be used and have been removed in Python 3."

The google python style guide is even more extreme, and does not allow relative imports at all (see section 2.3 of the [google style guide](https://google.github.io/styleguide/pyguide.html)).

Reply:
I see. I can see how you might say our import line is pretty long/verbose to make relative preferred here, but also it's not that long, and given that the absolute import is preferred - glad I asked.

Reply:
In DM we are more than happy to use relative imports within a single package...